setup.py

```diff
@@ -10,11 +10,26 @@
 # pylint: disable=wrong-import-order
 """Setup script for aiida-core package."""
 import json
+import sys
 import os
 
 from utils import fastentrypoints  # pylint: disable=unused-import
 from setuptools import setup, find_packages
 
+if (sys.version_info.major, sys.version_info.minor) == (3, 5):
+    import setuptools
+    from distutils.version import StrictVersion
+
+    REQUIRED_SETUPTOOLS_VERSION = StrictVersion('38.2.0')
+    INSTALLED_SETUPTOOLS_VERSION = StrictVersion(setuptools.__version__)
+
+    if INSTALLED_SETUPTOOLS_VERSION < REQUIRED_SETUPTOOLS_VERSION:
+        raise RuntimeError(
+            'The installation of AiiDA with Python version 3.5, requires setuptools>={}; your version: {}'.format(
```

Comment:
Should this rather be "installation of aiida-core" ?

Reply:
it's fine as is